pom.xml

```diff
@@ -39,6 +45,14 @@
                     <reportsDirectory>${basedir}/target/test_out</reportsDirectory>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>15</source>
```

Comment:
As a property for source/target versions, you have 2 more lines (line 12, 13) I would suggest removing mentioned lines

Reply:
Addressed in e7a4f19.